aws/rust-runtime/aws-config/Cargo.toml

```diff
@@ -9,7 +9,7 @@ license = "Apache-2.0"
 repository = "https://github.com/awslabs/smithy-rs"
 
 [features]
-client-hyper = ["aws-smithy-client/client-hyper"]
+client-hyper = ["aws-smithy-client/client-hyper", "aws-smithy-runtime/connector-hyper"]
```

Comment:
The connector type on Config is still from aws-smithy-client, so it has to stay here for now. My next PR will remove it since it is cutting over to the newer traits for config.